brooklyn-tosca-transformer/src/test/java/io/cloudsoft/tosca/a4c/brooklyn/plan/ToscaTypePlanTransformerIntegrationTest.java

```diff
@@ -70,6 +75,8 @@
 
 public class ToscaTypePlanTransformerIntegrationTest extends Alien4CloudIntegrationTest {
```

Comment:
This class is getting  quite big. Maybe we should consider splitting it based on scope (negative/positive tests - the samples could be grouped too) or by any other category.

Reply:
agree -- but not here